bigtable-client-core-parent/bigtable-client-core/pom.xml

```diff
@@ -204,13 +182,11 @@ limitations under the License.
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <version>${junit.version}</version>
-            <scope>test</scope>
```

Comment:
I'm a bit on the fence about depending on the bom for scopes. How would you feel about keeping them?

Reply:
Keeping it seems like a good idea.  I'll revert this change.